js/state/siteUtil.js

```diff
@@ -372,7 +372,13 @@ module.exports.filterSitesRelativeTo = function (sites, relSite) {
  * @param sites The application state's Immutable sites list.
  */
 module.exports.clearSitesWithoutTags = function (sites) {
```

Comment:
Pls update the name of this function and comment because it's a bit misleading. I'll merge this but just commit something directly with an auditor: @bbondy message. 
It filters out sites without tags, but it also clears last accessed time on the sites that do have tags.
I'd maybe call it clearHistory and then describe both actions in the comment.